backend/no_waste.py

```diff
@@ -52,9 +76,13 @@ def recipe(recipe_id: int):
     recipe = Recipe.query.get_or_404(recipe_id)  # get elem by id
 
     return jsonify(
-        {"id": recipe.id, "recipe_name": recipe.recipe_name, "recipe": recipe.recipe,}
+        {"id": recipe.id, "recipe_name": recipe.recipe_name, "recipe": recipe.recipe, }
```

Comment:
Why the format here is different from `/recipes` endpoint? I think we should use the serialize method after you implemented it already.